client/flutter/lib/generated/intl/messages_en.dart

```diff
@@ -23,12 +23,12 @@ class MessageLookup extends MessageLookupByLibrary {
-    "childFacts" : MessageLookupByLibrary.simpleMessage("Provide facts about what has happened, explain what is going on now and give them clear information about how to reduce their risk of being infected by the disease in words that they can understand depending on their age.\nThis also includes providing information about what could happen in a re-assuring way (e.g. a family member and/or the child may start not feeling well and may have to go to the hospital for some time so doctors can help them feel better)."),
-    "childLove" : MessageLookupByLibrary.simpleMessage("Children need adults\' love and attention during difficult times. Give them extra time and attention.\nRemember to listen to your children, speak kindly and reasure.\nIf possible, make opportunities for the child to play and relax."),
+    "childDistance" : MessageLookupByLibrary.simpleMessage("Try and keep children close to their parents and family and avoid separating children and their caregivers to the extent possible. If separation occurs (e.g. hospitalization) ensure regular contact (e.g. via phone) and reassurance."),
+    "childFacts" : MessageLookupByLibrary.simpleMessage("Provide facts about what has happened, explain what is going on now and give them clear information about how to reduce their risk of being infected by the disease in words that they can understand depending on their age.\nThis also includes providing information about what could happen in a reassuring way (e.g. a family member and/or the child may start not feeling well and may have to go to the hospital for some time so doctors can help them feel better)."),
+    "childLove" : MessageLookupByLibrary.simpleMessage("Children need adults\' love and attention during difficult times. Give them extra time and attention.\nRemember to listen to your children, speak kindly and reassure.\nIf possible, make opportunities for the child to play and relax."),
     "childRoutines" : MessageLookupByLibrary.simpleMessage("Keep to regular routines and schedules as much as possible, or help create new ones in a new environment, including school/learning as well as time for safely playing and relaxing."),
-    "childStress" : MessageLookupByLibrary.simpleMessage("Children may respond to stress in different ways such as being more clingy, anxious, withdraw, angry or agitated, bedwetting etc\nRespond to your child\'s reactions in a supportive way, listen to their concerns and give them extra love and attention."),
-    "cougningAndSneezing" : MessageLookupByLibrary.simpleMessage("When coughing and sneezing cover mouth and nose with flexed elbow or tissue"),
+    "childStress" : MessageLookupByLibrary.simpleMessage("Children may respond to stress in different ways such as being more clingy, anxious, withdrawn, angry or agitated; by bed-wetting etc\nRespond to your child\'s reactions in a supportive way, listen to their concerns and give them extra love and attention."),
```

Comment:
I think we can leave it "bedwetting" (https://books.google.com/ngrams/graph?content=bedwetting%2C+bed-wetting&year_start=1800&year_end=2008&corpus=15&smoothing=3&share=&direct_url=t1%3B%2Cbedwetting%3B%2Cc0%3B.t1%3B%2Cbed%20-%20wetting%3B%2Cc0#t1%3B%2Cbedwetting%3B%2Cc0%3B.t1%3B%2Cbed%20-%20wetting%3B%2Cc0)

```suggestion
    "childStress" : MessageLookupByLibrary.simpleMessage("Children may respond to stress in different ways such as being more clingy, anxious, withdrawn, angry or agitated; by bedwetting etc\nRespond to your child\'s reactions in a supportive way, listen to their concerns and give them extra love and attention."),
```

Reply:
thanks!

Reply:
undone